### PR TITLE
Fix gain/loss calculations on performance page

### PR DESCRIPTION
**Pull request by emaG3m**

### CODE REVIEW

File: app/reducers/performance.js [hunk 1]
```diff
@@ -156,8 +152,15 @@ const yesNoMarketDataLoadedReducer = (state, action) => {
       totals.noLongPotentialProfit = noLongProfit
       totals.yesPotentialProfit = yesShortProfit + yesLongProfit
       totals.noPotentialProfit = noShortProfit + noLongProfit
-      totals.yesGainLoss = totals.yesPotentialProfit - totals.yesCostBasis
-      totals.noGainLoss = totals.noPotentialProfit - totals.noCostBasis
+
+      if (totals.currentyesCollateralRisked > 0) {
```
Comment (emaG3m) [inline app/reducers/performance.js]: there is no currentGainLoss (meaning gains/losses are currently positioned in market) if you have no collateral at risk currently

**Comment by emaG3m** [comment]
fixes #115 among other things.

File: app/reducers/computed/sumPerformanceCalc.js [hunk 2]
```diff
@@ -1,10 +1,11 @@
 import _ from 'lodash'
 
-export default (prop, trader, performance) => (
+export default (prop, trader, performance, winningMarkets = {}) => (
```
Comment (mikec) [inline app/reducers/computed/sumPerformanceCalc.js]: might be simpler to pass in just the winning market type here? could use https://github.com/levelkdev/futarchy-app/blob/e34e557c8678dc4497cfe44ac04b1b8681db9575/app/constants/decisionMarketTypes.js and convert to lowercase

File: app/reducers/performance.js [hunk 3]
```diff
@@ -156,8 +152,8 @@ const yesNoMarketDataLoadedReducer = (state, action) => {
       totals.noLongPotentialProfit = noLongProfit
       totals.yesPotentialProfit = yesShortProfit + yesLongProfit
       totals.noPotentialProfit = noShortProfit + noLongProfit
-      totals.yesGainLoss = totals.yesPotentialProfit - totals.yesCostBasis
-      totals.noGainLoss = totals.noPotentialProfit - totals.noCostBasis
+      totals.yesGainLoss = totals.yesPotentialProfit - totals.currentyesCollateralRisked
```
Comment (mikec) [inline app/reducers/performance.js]: the case should be `currentYesCollateralRisked`

**Comment by mikec** [comment]
@emaG3m nice job with the tests! a couple comments..

File: app/reducers/computed/sumPerformanceCalc.js (hunk 2 above)
Comment (emaG3m) [reply app/reducers/computed/sumPerformanceCalc.js]: I'm not sure what you're implying here? My `winningMarket` obj is a mapping of all decision to whether or not they won.

Comment (mikec) [reply app/reducers/computed/sumPerformanceCalc.js]: oh yeah, I understand now - nevermind the comment

**Pull request merged**